ui/app/routes/vault/cluster/secrets/backend/configuration/edit.ts

```diff
@@ -82,6 +103,16 @@ export default class SecretsBackendConfigurationEdit extends Route {
     return model;
   }
 
+  isAzureConfigSet(response: Record<string, unknown>) {
```

Comment:
Just wondering if there's a more reliable way we could do this 🤔 why do we need to check if the config is set or not? In PKI for example, the backend always returns 200 so we never use `createRecord` we only ever query the record and then submit updates that way. I know there's some `isNew` logic in the template...but I'm wondering if we can refactor that and rely on something else. It looks like in the form we're already check for particular attributes so I'm not sure we need check for config here just to determine whether the record is new or not.

Maybe this would be easier to zoom about! haha